glean-core/android/src/test/java/mozilla/telemetry/glean/private/TimingDistributionMetricTypeTest.kt

```diff
@@ -182,6 +182,17 @@ class TimingDistributionMetricTypeTest {
         assertEquals(1L, snapshot.values[984625593])
         assertEquals(1L, snapshot.values[1969251187])
         assertEquals(1L, snapshot.values[2784941737])
+
+        // Assure the single sample API properly records.
+        metric.accumulateSingleSample(4L)
+
+        // Check that this new data was properly recorded in the second ping.
+        val snapshotTwo = metric.testGetValue("store1")!!
```

Comment:
This is just a nit, but you could have reused the `snapshot` above by changing it from a `val` to a`var`.  This is perfectly fine as is, I'm not too worried about a new instantiation of a variable in a test by any means :)